main.tex

```diff
@@ -419,12 +426,12 @@ \subsection{The l3build package}
 Both our framework and l3build support the updating of test files~\cite[Section~2.7]{latex2023l3build}. This allows developers to automatically generate parts of test files when they make fundamental changes to the code or when they develop complex new features, as discussed in Section~\ref{sec:updating-test-files}.
 
 \subsection{Batching and batch splitting}
-The techniques of batching and batch splitting were perhaps first used with \TeX{} in the ARQMath competitions in large-scale indexing of math formulae.
+The techniques of batching and batch splitting were perhaps first used with \TeX{} in the \ARQMath\ competitions in large-scale indexing of math formulae.
 
-In the first ARQMath competition, the \acro{MIRMU} team used the \LaTeX ML tool to convert \TeX{} formulae to the \acro{XML} format~\cite[Section~2.2]{novotny2020three}. Due to speed issues when processing each formula separately, \acro{MIRMU} processed them in batches. However, a single error would cause the loss of an entire batch. Therefore, \acro{MIRMU} used batch splitting to recover correct formulae after an error~\cite{novotny2020arqmath}.
+In the first \ARQMath\ competition, the \acro{MIRMU} team used \LaTeX\acro{ML} to convert \TeX{} formulae to \acro{XML}~\cite[Section~2.2]{novotny2020three}. Due to speed issues when processing each formula separately, \acro{MIRMU} processed them in batches. However, a single error would cause the loss of an entire batch. Therefore, \acro{MIRMU} used batch splitting to recover correct formulae after an error~\cite{novotny2020arqmath}.
```

Comment:
Since this is the first occurence of LaTeXML in the text, let's help the reader by qualifying LaTeXML as "a tool". Ommitting it does not seem to improve the line breaks.
```suggestion
In the first \ARQMath\ competition, the \acro{MIRMU} team used the \LaTeX\acro{ML} tool to convert \TeX{} formulae to \acro{XML}~\cite[Section~2.2]{novotny2020three}. Due to speed issues when processing each formula separately, \acro{MIRMU} processed them in batches. However, a single error would cause the loss of an entire batch. Therefore, \acro{MIRMU} used batch splitting to recover correct formulae after an error~\cite{novotny2020arqmath}.
```